docs/kubectl.md

```diff
@@ -390,6 +390,7 @@ Additional help topics:
   kubectl run-container Run a particular image on the cluster.
   kubectl stop          Gracefully shutdown a resource
   kubectl expose        Take a replicated application and expose it as Kubernetes Service
+  kubectl label         Update the labels on a resource
```

Comment:
I'm fine with that, so long as we're consistent with the pattern. We're probably going to need a kubectl-conventions.md.